src/main/java/com/github/zeger_tak/enversvalidationplugin/validate/AuditTableInformationMapValidator.java

```diff
@@ -57,4 +62,28 @@ public void validateAllExistingAuditTablesAreSpecified()
 			throw new ValidationException("The following audit tables are not configured in the audit table information map: " + auditTablesNotSpecified);
 		}
 	}
+
+	@Validate
+	public void validateAllContentTablesHaveAllColumnsInAuditTable() throws DataSetException, SQLException
```

Comment:
Wordt dit niet afgevangen door com.github.zeger_tak.enversvalidationplugin.validate.RevisionValidator#validateAllRecordsInContentTableHaveAValidLatestRevision ?

"The content table may not have columns which are not present in the audit table."

Even kort gekeken naar de code in die RevisionValidator en die vangt sowieso geen missende columns als er geen data in de content table zit...

Reply:
`com.github.zeger_tak.enversvalidationplugin.validate.RevisionValidator#validateAllRecordsInContentTableHaveAValidLatestRevision` gaat volgens mij over data ín de table, `validateAllContentTablesHaveAllColumnsInAuditTable` (zeg maar het doel van dit PR) gaat over missende kolommen (structuur).

In ieder geval worden missende audit kolommen niet afgevangen, want daar liepen we dus tegenaan. Vandaar dit pull request :)